### ISSUE TITLE
brew users forced to install glide

### ISSUE THREAD

**Issue by y0ssar1an** [junegunn/fzf#1249]
A few weeks ago, I submitted [a PR to vendor all of `fzf`'s dependencies](https://github.com/junegunn/fzf/pull/1218). It was rejected for [this reason](https://github.com/junegunn/fzf/pull/1218#issuecomment-372560468):
>most users of fzf simply download the official binaries using the install script rather than to build it from the source. For them, adding vendor directory will only make cloning slower.

Today, I went to install `fzf` through homebrew, and noticed this isn't the case.
```sh
% brew info fzf                                                                                                                                                                                       0
fzf: stable 0.17.3 (bottled), HEAD
Command-line fuzzy finder written in Go
https://github.com/junegunn/fzf
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/fzf.rb
==> Dependencies
Build: glide ✘, go ✔
```
If you look at [the Homebrew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/fzf.rb#L23), you can see that it's running `glide install`.

If a sizeable fraction of your users installing with `brew`, then a sizeable fraction of your users are building from source. Most of them are not Go programmers and don't want a Go vendoring tool installed on their systems. Even if they don't mind `glide` being installed, they're still wasting time fetching, compiling, and running `glide`. **There's no reason to make them do this**. Also, since the user's version of `glide` is managed by `brew`, there's a chance an update to `glide` could break builds of `fzf`.

In summary, forcing users to fetch dependencies at build time 1) is slow, 2) adds failure cases, and 3) leaves behind unwanted software on the user's system. Please consider vendoring your dependencies and eliminating the dependency on `glide`.

**Comment by junegunn** [comment]
> If a sizeable fraction of your users installing with brew, then a sizeable fraction of your users are building from source.

That is not true, I think you're mistaken.

```
> brew info fzf
fzf: stable 0.17.3 (bottled), HEAD
Command-line fuzzy finder written in Go
https://github.com/junegunn/fzf
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/fzf.rb
==> Dependencies
Build: glide ✘, go ✘
==> Options
--HEAD
        Install HEAD version
==> Caveats
To install useful keybindings and fuzzy completion:
  /usr/local/opt/fzf/install

To use fzf in Vim, add the following line to your .vimrc:
  set rtp+=/usr/local/opt/fzf

> brew list | grep ^go$

> brew list | grep ^glide$

> brew install fzf
==> Downloading https://homebrew.bintray.com/bottles/fzf-0.17.3.high_sierra.bottle.tar.gz
Already downloaded: /Users/jg/Library/Caches/Homebrew/fzf-0.17.3.high_sierra.bottle.tar.gz
==> Pouring fzf-0.17.3.high_sierra.bottle.tar.gz
==> Caveats
To install useful keybindings and fuzzy completion:
  /usr/local/opt/fzf/install

To use fzf in Vim, add the following line to your .vimrc:
  set rtp+=/usr/local/opt/fzf
==> Summary
🍺  /usr/local/Cellar/fzf/0.17.3: 17 files, 3.2MB

> brew list | grep ^go$

> brew list | grep ^fzf$
fzf

>
```

**Comment by y0ssar1an** [comment]
You are correct. I misunderstood the Homebrew formula. I'm sorry for the noise.

**Comment by junegunn** [comment]
No problem, thanks for the suggestion though.
